src/lang/de-DE.json

```diff
@@ -537,6 +537,8 @@
   "skill_1": "Normale",
   "skill_2": "Hohe",
   "skill_3": "Sehr hohe",
+  "show_more_details": "Mehr Details",
```

Comment:
adding strings to this file is not necessary, translations come in via crowdin in a later step.